### PR TITLE
Add precisions for developers on README

### PR DESCRIPTION
**Pull request by vprigent**
Because well, seems stupid only when you know it

### CODE REVIEW

**Comment by KeitIG** [comment]
Well, in this case, could you add a list of steps to set up Electron with Museeks ? (It's true it's not really clear)

**Comment by vprigent** [comment]
This should do the job.

We could perhaps add some links for good practice guides ?
However I have none for electron/node/react

File: README.md [hunk 1]
```diff
@@ -89,10 +89,14 @@ If you want to report a bug, first, thanks a lot. To help us, please indicate yo
 
 - Fork and clone
 - Master is usually unstable, checkout to a tag to have a stable state of the app
-- `npm install && npm run dev`
-- Enable dev mode in the app in the settings view to show DevTools
 
-Webpack will watch for JSX and SASS changes.
+- Install the latest version of electron by running `npm install -g electron`
+- NPM will install electron globally. You can use it on every of your electron projects now.
```
Comment (YurySolovyov) [inline README.md]: > You can use it on every of your electron projects now.

I'd rephrased that. Latest electron might not work for just about any electron project.
Something like "You can use electron globally now with `electron [appdir]` command." or something like that,

Comment (KeitIG) [inline README.md]: I'd put the manual electron install too. I don't like to use global package for that.

My recommended way is to download Electron, then place the app in `resources/app`.

**Comment by vprigent** [comment]
I read what I could about the manual electron. Never used that way before and never would again, it's a lot more painful as a beginner, even if it's safer

**Comment by KeitIG** [comment]
> it's a lot more painful as a beginner, even if it's safer

That's the point.

Otherwise it's fine too me, thanks for that.
